README.md

```diff
@@ -48,6 +48,58 @@ docker run --rm \
   ghcr.io/oviva-ag/epa4all-rest-service:latest
 ```
 
+### docker-compose with userspace wireguard
+This would be an example on how to run the epa4all-rest-service with a VPN connection to the Konnektor. The VPN connection is established via a userspace wireguard implementation. The service uses the network stack of the VPN container.
+
+Note: a userspace wireguard implementation is not as performant as a kernel implementation. If you want to use a kernel implementation, you need to run the VPN container in host network mode.
+```yaml
+services:
+  epa4all-rest:
+    container_name: epa4all-rest
+    image: epa4all-rest-service:latest
+    ports:
+        - '127.0.0.1:8099:8080'
+    volumes:
+        - ./vKon_Client_XXX.p12:/credentials.p12
+    environment:
+        - EPA4ALL_KONNEKTOR_URI=https://10.156.145.103:443
+        - EPA4ALL_ENVIRONMENT=RU
+        - EPA4ALL_PROXY_ADDRESS=127.0.0.1
+        - EPA4ALL_KONNEKTOR_PROXY=false
```

Comment:
Instead of a flag, why not only use a proxy when its set?

I.e. if `EPA4ALL_PROXY_ADDRESS` is empty, no proxy is used.

Reply:
That was my initial intention as well. At that point, I was convinced that (some) sort of proxy was still needed in the `Epa4allClientService.java` part. That was wrong. On a second try, I got it to work.

Now simply omitting the `EPA4ALL_PROXY_ADDRESS` will not add a proxy to the http client 👍

Reply:
Also added a comment about GKE to the README!